website/docs/r/is_vpc_routing_table.html.markdown

```diff
@@ -41,6 +41,7 @@ resource "ibm_is_vpc_routing_table" "test_ibm_is_vpc_routing_table" {
 ## Argument reference
 Review the argument references that you can specify for your resource. 
 
+- `created_at` - (Timestamp)  The date and time that the default routing table was created.
```

Comment:
not default. 
The date and time that this routing table was created



<br class="Apple-interchange-newline">